tests/foreman/ui/test_gpgkey.py

```diff
@@ -1,9 +1,9 @@
 # -*- encoding: utf-8 -*-
 """Test class for GPG Key UI"""
 
+import random
```

Comment:
What that said. And yes, once the module is imported it is entirely on the memory. Import a specific name is just a matter of making short to call it.

But I prefer to use the stdlibs modules with the import statement since it will show in the code (without the need of going back to imports section) from where some name is coming from.